BUILD.md

```diff
@@ -176,7 +206,8 @@ ctest -C Debug --output-on-failure --parallel 16
 ctest -C Release  --output-on-failure --parallel 16
 ```
 
-### Linux and macOS Build
+### Linux AMD64, Linux ARM64 and macOS Build
+For Linux ARM64 build, run the following commands in [chroot sandbox environment](BUILD.md#linux-arm64-additional-system-requirements).
```

Comment:
This makes if seem like the commands below are are only for ARM64 and there are no instructions for AMD64/MacOS.

Suggest moving this sentence below the code block and change it to: 

```suggestion
For Linux ARM64 build, run the above commands in the [chroot sandbox environment](BUILD.md#linux-arm64-additional-system-requirements).
```